### DISCUSSION TITLE
Is librsvg really licensed under GPL?

### DISCUSSION THREAD

**Discussion by keven-ma** [dlemstra/Magick.NET#1482] [General] [answered]
librsvg v2.40 (https://github.com/GNOME/librsvg/tree/librsvg-2.40) contains two license files. One is GPL and the other is LGPL. According to https://gitlab.gnome.org/GNOME/librsvg/-/issues/72 and https://gitlab.gnome.org/GNOME/librsvg/-/issues/432. Federico Mena Quintero (the maintainer of librsvg) said the license should be LGPL. So can we change the [LICENSE] in (https://github.com/ImageMagick/VisualMagick/blob/main/rsvg/Config.txt) to librsvg\COPYING.LIB?

**Comment by dlemstra** [comment, marked as answer]
We can only use the LGPL license so that seems to be a mistake in the config.txt file from my side. I have updated the config file that you mention here. Thanks for pointing this out.

**Comment by keven-ma** [comment]
GPL is right. 

https://github.com/ImageMagick/rsvg has two library librsvg (@v2.40.x) (https://github.com/ImageMagick/rsvg/tree/main/librsvg) and [gdk-pixbuf] (@v2.36.x) (https://github.com/ImageMagick/rsvg/tree/main/gdk-pixbuf). 
librsvg is indeed LGPL. But gdk-pixbuf is GPL. gdk-pixbuf changed to LGPL at v2.37.0.
Once https://github.com/ImageMagick/rsvg/tree/main/gdk-pixbuf update to v2.37.0+ we can update the config file.

**Reply by dlemstra** [reply]
The COPYING file from gdk-pixbuf is already LGPL?

**Reply by keven-ma** [reply]
gdk-pixbuf v2.37.0 is LGPL.  But gdk-pixbuf in ImageMagick is v2.36.x. It's GPL.

**Reply by dlemstra** [reply]
Everything I am reading says it is LGPL. Why do you think they use GPL in older versions?

**Reply by keven-ma** [reply]
I checked this file https://github.com/GNOME/gdk-pixbuf/blob/gdk-pixbuf-2-36/COPYING. It's GNU GPLv2.

**Reply by keven-ma** [reply]
This file (https://github.com/ImageMagick/rsvg/blob/main/gdk-pixbuf/COPYING) is also GNU GPL v2

**Reply by keven-ma** [reply]
The license is changed to LGPL in https://github.com/GNOME/gdk-pixbuf/blob/2.37.0/COPYING. 
Can we update gdk-pixbuf to v2.37.0+? After that the whole rsvg project is LGPL.

**Reply by dlemstra** [reply]
Version `2.36.0` uses version `2.0` of the `LGPL` that used to be called `GNU LIBRARY GENERAL PUBLIC LICENSE` and version `2.37.0` uses version `2.1` of the `LGPL` that is now called `GNU LESSER GENERAL PUBLIC LICENSE`. So I don't think there is a need to upgrade the library.
